numpy/distutils/command/config_compiler.py

```diff
@@ -15,7 +17,45 @@ def show_fortran_compilers(_cache=None):
     dist = distutils.core._setup_distribution
     show_fcompilers(dist)
 
-class config_fc(Command):
+class config(Command, metaclass=abc.ABC):
```

Comment:
While distutils uses lower-case classes, I think we should use Upper-case. This will require more changes through the file ...

```suggestion
class Config(Command, metaclass=abc.ABC):
```